rep-0146.rst

```diff
@@ -0,0 +1,80 @@
+REP: 146
+Title: ROS Robot Parameter Naming Convention
+Author: Kei Okada <k-okada@jsk.t.u-tokyo.ac.jp>
+Status: Draft
+Type: Informational
+Content-Type: text/x-rst
+Created: 31-Jly-2015
+Post-History:
+
+Abstract
+========
+
+This REP aims to define the best practices for robot related parameter
+name space for making robot-agnostic packages.
+
+
+Motivation
+==========
+
+As the number of ROS-supported robot increases, we will write
+robot-agnostic packages, but still they need robot-specific
+information as calibration parameters.
+
+A `robot_description` parameter is widely shared naming convention
+among ROS community and this REP proposes such conventions for robot specific information on
+parameter, so application program can build their program on top of that.
+
+
+Specifications
+==============
+
+Parameters
+----------
+
+robot_description
+'''''''''''''''''
+
+`robot_description` stores robot kinematics / geometry information using
+URDF format.
+
+
+robot_type
+''''''''''
+
+`robot_type` stores robot type in lower cases, for example 'pr2' in PR2 robot case [1], 'pepper' in Pepper robot case and 'fetch' in Fetch robot cases. This could be regarded as manufacturer's model name.
+
+
+robot_name
+''''''''''
+
+`robot_name` stores robot name, default pr2 name is 'robot' [1], which is given individual name to the robot. So robot name of 'pr2' robot type could be 'james', 'gutsy' etc...
+
+robot/info
+''''''''''
+
+`robot_info` stores any other characteristics of the robot in YAML dict type, including type, serial number, owner name / email address, timezone, language it uses ...
```

Comment:
You changed one of the two instances of `robot/info` to `robot_info` but I still think that if we're not going to define any schema there's no point in including it in this standard. That's not to say it can't be there, but until we can desctibe what goes inside it then we shouldn't try to add it here. We can always amend/extend/replace this in the future when it's clear what should be in that field.

Reply:
ok, removed.